src/rrt/Tree.hpp

```diff
@@ -1,15 +1,15 @@
 #pragma once
 
-#include <deque>
 #include <flann/algorithms/dist.h>
 #include <flann/algorithms/kdtree_single_index.h>
+#include <stdlib.h>
+#include <deque>
 #include <flann/flann.hpp>
 #include <functional>
 #include <list>
 #include <memory>
 #include <rrt/StateSpace.hpp>
 #include <stdexcept>
-#include <stdlib.h>
```

Comment:
What's the thought behind re-ordering the includes?

Reply:
Stylize requested it.